sormas-api/src/main/java/de/symeda/sormas/api/caze/CaseDataDto.java

```diff
@@ -346,6 +346,7 @@ public class CaseDataDto extends SormasToSormasShareableDto implements IsCase {
 	@Valid
 	@EmbeddedPersonalData
 	@EmbeddedSensitiveData
+	@SensitiveData
```

Comment:
Shouldn't this and the other fields be sensitive only for luxembourg servers?

Reply:
I remember we discussed on the planning that if we make the health conditions on the case sensitive, then it has to be sensitive on all other places it is used